### PR TITLE
Production

### PR DESCRIPTION
**Pull request by tresoldi**
This PR updates the tutorial for publication. Main changes:

- depending on the latest `lingpy`
- using the new (pylexibank2.1) dataset
- minor grammar changes/fixes

### CODE REVIEW

**Review by LinguList — COMMENTED** [review]
Looks fine to me, but one question: the cldf-new here, is it based on the output from lingpy? If so, we should file an issue forcing lingpy to modify the policty with identifiers, as it goes no against our lexibank principles, right?

**Comment by chrzyki** [comment]
Please keep this open a little bit longer if you don't mind. I'll only have time to play around with this tomorrow or on Friday. Thanks!

**Comment by tresoldi** [comment]
@chrzyki sure, no problem.

@LinguList yes, `cldf-new` is generated from lingpy, with the identifiers coming from the local data in `cldf-khobwa`. The latter is not generated from `lieberherrkhobwa`, but was manually derived the first time (removing items, languages, etc.). We also remember users, when showing how to load a Lexibank dataset, that the data is different and, as such, the results will be different.

`lingpy` is doing the right thing, in my opinion: we strongly recommend that people use Lexibank, but as long as the identifiers are valid CLDF identifiers they should be accepted.

File: Tutorial.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Tutorial accompanying the chapter "Managing historical linguistic data for computational phylogenetics and computer-assisted language comparison"
  
-# Tresoldi, Tiago; Rzymski, Christoph; Forkel, Robert; Greenhill, Simon J.; List, Johann-Mattis
+# Tresoldi, Tiago; Rzymski, Christoph; Forkel, Robert; Greenhill, Simon J.; List, Johann-Mattis; Gray, Russel D.
```
Comment (SimonGreenhill) [inline Tutorial.md]: Russell

File: Tutorial.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-This tutorial supplements the aforementioned chapter, intentionally following a similar progression of analysis. The aim is to facilitate understanding of the more theoretical topics of the chapter while allowing readers to see step-by-step examples of the input, intermediate, and output files of a common phylogenetic analysis.
+This tutorial supplements the aforementioned chapter, intentionally following a similar progression of analysis. The aim is to facilitate the understanding of the more theoretical topics of the chapter while allowing readers to see step-by-step examples of the input, intermediate, and output files of a common phylogenetic analysis.
```
Comment (SimonGreenhill) [inline Tutorial.md]: "chapter, intentionally following" -> "chapter and intentionally follows"

File: Tutorial.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-The workflow uses the Python programming language and a number of libraries developed at the Max-Planck-Institute for the Science of Human History (MPI-SHH), in particular the `lingpy` library. All these libraries are free and open source. For more information on LingPy, we recommend readers see the tutorial published as [supplementary material](https://github.com/lingpy/lingpy-tutorial) to "[Sequence comparison in computational historical linguistics](https://academic.oup.com/jole/article/3/2/130/5050100)" by List et al. (2018). 
+The workflow uses the Python programming language and several libraries developed at the Max-Planck-Institute for the Science of Human History (MPI-SHH), in particular the `lingpy` library. All these libraries are free and open source. For more information on LingPy, we recommend readers see the tutorial published as [supplementary material](https://github.com/lingpy/lingpy-tutorial) to "[Sequence comparison in computational historical linguistics](https://academic.oup.com/jole/article/3/2/130/5050100)" by List et al. (2018).
```
Comment (SimonGreenhill) [inline Tutorial.md]: "Max-Planck-Institute" -> "Max Planck Institute"

File: Tutorial.md (hunk 1 above)
Comment (tresoldi) [reply Tutorial.md]: Wow... This was embarrassing.

File: Tutorial.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+We strongly recommend using virtual environments for reproducing the steps outlined here. Virtual environments are a practical solution for creating independent configurations for testing and experimenting, with no interference on system-wide installation. More information on virtual environments can be found on the [Python documentation](https://docs.python.org/3/tutorial/venv.html). In most systems, a virtual environment can be created by calling `python -m venv env`. Once created, the environment can be activated with `source env/bin/activate` and deactivated with the `deactivate` command.
```
Comment (SimonGreenhill) [inline Tutorial.md]: "experimenting, with no interference on system-wide installation." -> "experimenting, that do not interfere with any other python programs installed on the system"

File: Tutorial.md [hunk 5]
```diff
@@ -38,9 +38,9 @@ As this tutorial is intended to illustrate the management of data for phylogenet
...
+While the details and advantages of CLDF have been discussed and presented in Forkel et al. (2018), we would like to highlight the benefits of using CLDF as a basis for datasets in historical linguistics and, by extension, phylogenetic analyses. While, at a first glance, CLDF and the outer appearance of a dataset (e.g. information split across multiple files), might seem daunting or even plainly unnecessary, it gives researchers a powerful toolset via the expressiveness of relational data structures in well-supported serialization formats, namely CSV and JSON. The format allows for the creation of truly linked data (see for example the embedded information concerning languoids and concepts in the Kho-Bwa sample dataset), which at the same time exists independently of any other software stack, given that all the information are stored in plain (but structured) text files. The fact that multiple files are 'glued' together by a JSON file, which specifies how files are related as well as metadata information for every file and column, allows for unique identifiers for every record, and guarantees maximum flexibility regarding research designs and experimental setups.
```
Comment (SimonGreenhill) [inline Tutorial.md]: While the details and advantages of CLDF have been discussed and presented in Forkel et al.
(2018), we would like to highlight the benefits for historical linguistics and, by extension,
phylogenetic analyses. 
While at first glance CLDF might seem daunting or unnecessary, it has many benefits. First,
the structure is easily importable into standard statistical and analytic tools in many
programming languages or analysis packages. Second, CLDF gives researchers a powerful toolset
to express relational data structures in well-supported open formats, namely CSV and JSON. 
Third, the metadata contained within CLDF describes the data structure for enabling permanent
documentation of the data structure, and facilitating the re-use of the data. Further, the
metadata allows known types of columns to be validated and checked for incorrect content
ensuring that the data is correct. Fourth, CLDF is linked data which means that any type of
extra information can be easily added to the dataset by linking it to the relevant identifier 
(see for example the embedded information concerning languoids and concepts in the Kho-Bwa
sample dataset). These data exist independently of any other software stack as all they can
be stored in plain (but structured) text files. The fact
that multiple files are 'glued' together by a JSON file, which specifies how files are
related as well as metadata information for every file and column, allows for unique
identifiers for every record, and guarantees maximum flexibility regarding research designs
and experimental setups.

File: Tutorial.md [hunk 6]
```diff
@@ -109,7 +109,7 @@ ant  ant  Duhumbi  Duhumbi    kʰin-ʨʰɔk  kʰ i n + tɕʰ ɔ k          701
 ---  ---  -------  -------    ---------  ----------------          ---
 ```
 
-The statistics reported by the command-line `cldf` utility can be confirmed by inspecting the in-memory wordlist. Besides confirming the number of languages, concepts, and forms, we can extract more advanced statistics such as minimal and mutual average coverage of concepts across the languages in the dataset [@List2018f]. These statistics provide useful information that can help us ascertain if we have an acceptable amount of comparable material for a phylogenetic analysis.
+The statistics reported by the command-line `cldf` utility can be confirmed by inspecting the in-memory wordlist. Besides confirming the number of languages, concepts, and forms, we can extract more advanced statistics such as minimal and mutual average coverage of concepts across the languages in the dataset. These statistics provide useful information that can help us ascertain if we have an acceptable amount of comparable material for a phylogenetic analysis.
```
Comment (SimonGreenhill) [inline Tutorial.md:112]: /acceptable/ -> /sufficient/

File: Tutorial.md [hunk 7]
```diff
@@ -244,7 +244,7 @@ e       -       j       u       -
 e       -       j       u       -
 ```
 
-The alignment object is a wordlist which carries information from its source, and can written to disk as a single-table or CLDF dataset. The alignment can therefore be easily exported into other tools, for example Edictor (mentioned in #3.4), which is designed for exploration and manipulation of this kind of data. A tab-separated output, for example, can be generated with:
+The alignment object is a wordlist which carries information from its source, and can be written to disk as a single-table or CLDF dataset. The alignment can therefore be easily exported into other tools, for example Edictor (mentioned in #3.4), which is designed for exploration and manipulation of this kind of data. A tab-separated output, for example, can be generated with:
```
Comment (SimonGreenhill) [inline Tutorial.md:247]: "tab-separated output" -> "tab-separated output file"

File: Tutorial.md [hunk 8]
```diff
@@ -392,10 +392,10 @@ to_cldf(wl, path='cldf-new')
...
+Given the complexity of the topic, the current tutorial can only provide an incomplete introduction to the fascinating world of data management in computational phylogenetics and computer-assisted language comparison. We hope, however, to have shown enough to ease the entry for readers with little experience in data managment and phylogenetic analysis.  If you have questions on the management of linguistic data for phylogenetic analyses or CLDF, don't hesitate to contact us at <lexibank@shh.mpg.de>. Questions about LingPy can be posted either as an issue on its [GitHub repository](https://github.com/lingpy/lingpy) or sent to <info@lingpy.org>.
```
Comment (SimonGreenhill) [inline Tutorial.md]: "to the fascinating world of " -- a bit over the top :) Please delete.

File: pip-requirements.txt [hunk 9]
```diff
@@ -1,4 +1,4 @@
 tabulate
--e git+https://github.com/lingpy/lingpy.git@86c9f22ef347a4428ed99970c89d83fa8205dce5#diff-9584dcd0d5e720b72181112bb1b56c58#egg=lingpy
+lingpy
```
Comment (SimonGreenhill) [inline pip-requirements.txt]: should we specify a min-version of lingpy here?

Comment (tresoldi) [reply pip-requirements.txt]: Yes, good idea.

File: Tutorial.md (hunk 8 above)
Comment (tresoldi) [reply Tutorial.md]: Oh, come on, will you tell me you're not thrilled about it, that kids don't say "when I grow up, I want to manage data"? (seriously, deleting :wink:)

**Comment by chrzyki** [comment]
Looks good, thanks! Maybe it'd be worthwhile to spell out somewhere that everything inside `python` markdown has to be run *inside* of a Python environment/interpreter?

**Review by chrzyki — APPROVED** [review]

**Comment by tresoldi** [comment]
@chrzyki wouldn't hurt... I could also make a stand-alone script with everything in it, do you think it is worth it?

File: Tutorial.md [hunk 10]
```diff
@@ -21,7 +21,7 @@ pip install --upgrade wheel
 pip install -r pip-requirements.txt
 ```
 
-All the code snippets below assume that the necessary libraries have already been imported in Python. The full list of imports, which can be executed upon initialization and for testing that all libraries were properly installed, is:
+All the code snippets below must be executed from inside Python (either by calling `python` from the command line, entering them one by one, or by placing them in a Python script) and assume that the necessary libraries have already been imported in Python. The full list of imports, which can be executed upon initialization and for testing that all libraries were properly installed, is:
```
Comment (chrzyki) [inline Tutorial.md:24]: That's good and enough, imo!

Comment (tresoldi) [reply Tutorial.md:24]: Great. I'll merge and move to the DOCX.

**Pull request merged**